### PR TITLE
[cli/feature]: split feature into a separate file

### PR DESCRIPTION
**Pull request by lguohan**
Signed-off-by: Guohan Lu <lguohan@gmail.com>

**- What I did**
split feature into a separate file

**- How I did it**
create new feature.py for both config and show, move the code into feature.py
move common code into sonic-utiltities.common.cli.py

**- How to verify it**
unit test
also run it in vs switch

**- Previous command output (if the output of a command-line utility has changed)**

**- New command output (if the output of a command-line utility has changed)**

### CODE REVIEW

**Comment by lguohan** [comment]
will use similar approach to split other commands.

**Comment by jleveque** [comment]
>will use similar approach to split other commands.

FYI, this is a helpful start toward resolving https://github.com/Azure/sonic-utilities/issues/967!

File: config/main.py [hunk 1]
```diff
@@ -935,8 +899,7 @@ def config(ctx):
 
     ctx.obj = Db()
 
-pass_db = click.make_pass_decorator(Db, ensure=True)
-
+config.add_command(feature.feature)
```
Comment (jleveque) [inline config/main.py:902]: Suggest to move this after the `aaa` `add_command()` lines to keep them in alphabetical order, and add a newline after.

File: show/feature.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import click
+from natsort import natsorted
+from tabulate import tabulate
+
+from utilities_common.cli import pass_db
+
+#
+# 'feature' group (show feature ...)
+#
+@click.group(name='feature', invoke_without_command=False)
```
Comment (jleveque) [inline show/feature.py:10]: Add `cls=AbbreviationGroup`

Comment (lguohan) [reply show/feature.py:10]: how to test the abbreviations?

Comment (jleveque) [reply show/feature.py:10]: Type enough chars to distinguish the command from other commands and the subcommand from other subcommands. E.g., `show fea au`, `show fea st`, etc.

Comment (lguohan) [reply show/feature.py:10]:
```
admin@vlab-01:~$ show feat st
Feature         State     AutoRestart
--------------  --------  -------------
bgp             enabled   enabled
database        enabled   disabled
dhcp_relay      enabled   enabled
lldp            enabled   enabled
mgmt-framework  enabled   enabled
nat             disabled  enabled
pmon            enabled   enabled
radv            enabled   enabled
sflow           disabled  enabled
snmp            enabled   enabled
swss            enabled   enabled
syncd           enabled   enabled
teamd           enabled   enabled
telemetry       enabled   enabled
```

**Comment by lguohan** [comment]
@jleveque , rebased to latest master

**Pull request merged**